tutorials/vpn-interop-guides/using-cloud-vpn-with-aws/index.md

```diff
@@ -0,0 +1,591 @@
+
+Select the newly created VPC from the VPC Dashboard in order to collect the configuration detail required to complete the [GCP configuration](#heading=h.25npjme3proz):
+
+![image](cloudvpnguide---fppuk5jplxe.png)
+
+The last step is to collect the IP addresses of the AWS Virtual Gateway and the pre-shared keys used for IKE authentication that are automatically generated by AWS.  This information is stored in the _configuration file_ which you can download by clicking **Download Configuration**.  Although several device-specific options are available for the file configuration format,for GCP, select **Generic**:
+
+![image](cloudvpnguide---un801tai0ms.png)
+
+![image](cloudvpnguide---ng77v1fh7t.png)
+
+The configuration file is an ASCII text file.  Within the file, the auto-generated pre-shared key is listed under **Pre-Shared Key**.  
+
+A sample configuration file is provided below for reference. 
+
+NOTE:  AWS creates two VPN tunnels under "VPN connections" and there are two sets of VPN parameters listed in the sample configuration file, one set for each tunnel.  These parameters need to match the tunnel parameters on the GCP side that will be configured later in this document:
+
+```
+Amazon Web Services
+Virtual Private Cloud
+
+VPN Connection Configuration
+================================================================================
+AWS utilizes unique identifiers to manipulate the configuration of 
+a VPN Connection. Each VPN Connection is assigned a VPN Connection Identifier 
+and is associated with two other identifiers, namely the 
+Customer Gateway Identifier and the Virtual Private Gateway Identifier.
+
+Your VPN Connection ID		         : vpn-c1c6d9d3
+Your Virtual Private Gateway ID          : vgw-f670afe8
+Your Customer Gateway ID    		 : cgw-3548972b
+		
+A VPN Connection consists of a pair of IPSec tunnel security associations (SAs). 
+It is important that both tunnel security associations be configured. 
+
+IPSec Tunnel #1
+================================================================================
+#1: Internet Key Exchange Configuration
+		
+Configure the IKE SA as follows
+  - Authentication Method    : Pre-Shared Key 
+  - Pre-Shared Key           : auto-generated-pre-shared-key
+  - Authentication Algorithm : sha1
+  - Encryption Algorithm     : aes-128-cbc
+  - Lifetime                 : 28800 seconds
+  - Phase 1 Negotiation Mode : main
+  - Perfect Forward Secrecy  : Diffie-Hellman Group 2
+```
+
+> 
+## Configuration - GCP UI
+
+In the Google Cloud Platform Console, either select the GCP project into which the VPN will be deployed, or create a new project.  More information on creating and managing projects can be found [here](https://cloud.google.com/compute/docs/projects).    
+To create a new VPN , click on the main services menu located at the top left corner in the Console. Under **Networking**, select **[Hybrid Connectivity **and** VPN**](https://console.cloud.google.com/vpn/list):
```

Comment:
Remove space between VPN and comma.

In last sentence, I'm not sure how you are trying to format the link text. But, the asterisks are showing.